spyder/plugins/framesexplorer/widgets/main_widget.py

```diff
@@ -92,23 +92,23 @@ def setup(self):
         # ---- Options menu actions
         exclude_internal_action = self.create_action(
             FramesExplorerWidgetActions.ToggleExcludeInternal,
-            text=_("Exclude internal frames"),
+            text=_("Exclude internal frames from inspect"),
```

Comment:
```suggestion
            text=_("Exclude internal frames when inspecting execution"),
```

I think this would be clearer for users. However, if it's not what mean, please suggest a more telling text.